remove impteam logic; will be handled by keybase client

mmou opened the pull request:
blocked on keybase client release

xgess commented:
what do you think about updating the library to change
```python
res = await bot.kvstore.put(team, namespace, key, value, revision=None)
```
into
```python
res = await bot.kvstore.put(namespace, key, value, team="whatever", revision=None)
```
and we could update the simple storage example not to use a team at all. thoughts?

mmou commented:
i changed the kvstoreclient to make `team` an optional method arg.
i changed the simple storage example to use the default team.
i then made the minimal amount of changes to the two other examples to work with the kvstoreclient changes. given the impteam change, storing values in the bot's store (instead of the team's store) does make more sense, but the effort to remove explicit mention of team names feels too high for the payback at this time.

xgess commented on pykeybasebot/kvstore_client.py:
```diff
@@ -60,40 +61,39 @@ def __init__(self, bot):
-        res = await self.execute(
-            {"method": "list", "params": {"options": {"team": team}}}
-        )
+        args: Dict[str, Any] = {"method": "list", "params": {"options": {"team": team}}}
```
```suggestion
        args: Dict[str, Any] = {"method": "list", "params": {"options": {}}}
```
no?

mmou replied:
this made me realize that the kb chat API deals with null values fine
